server/index.js

```diff
@@ -17,6 +18,15 @@ app.get('/shop/:productId', (req, res) => {
   });
 });
 
+app.get('/shop/', (req, res) => {
```

Comment:
similar to what @elizabethcan said above, you need to change this endpoint to include the shopID.